js/services/CohortDefinition.js

```diff
@@ -151,17 +151,7 @@ define(function (require, exports) {
 	}
 
 	function getCohortPrintFriendly(cohortExpression) {
-		var getCohortPrintFriendlyPromise = $.ajax({
-			url: config.webAPIRoot + 'cohortdefinition/printfriendly/cohort?format=html',
-			method: 'POST',
-			contentType: 'application/json',
-			data: JSON.stringify(cohortExpression),
-			error: function (error) {
-				console.log("Error: " + error);
-				authApi.handleAccessDenied(error);
-			}
-		});
-		return getCohortPrintFriendlyPromise;
+		return httpService.plainTextService.doPost(config.webAPIRoot + 'cohortdefinition/printfriendly/cohort?format=html', cohortExpression);
```

Comment:
Thanks for updating this service. I initially tried to use the httpService (as per the other places in the file) but I coudln't figure out how to allow it to handle plain-text responses. I didn't realizet here was a `plainTextService` api of httpService.